test/regress/CMakeLists.txt

```diff
@@ -2163,6 +2159,12 @@ set(regress_1_tests
   regress1/sygus/max.sy
   regress1/sygus/max2-bv.sy
   regress1/sygus/multi-fun-polynomial2.sy
+  regress1/sygus/new/complex-no-rewrite.sy
```

Comment:
I'd prefer if these regressions were not in a new folder `new/`.

Reply:
Done.